README.md

```diff
@@ -1 +1,188 @@
-# sti-python
+Python for OpenShift - Docker images
+========================================
+
+This repository contains sources of the images for building various versions
+of Python applications as reproducible Docker images using
+[source-to-image](https://github.com/openshift/source-to-image).
+User can choose between RHEL and CentOS based builder images.
+The resulting image can be run using [Docker](http://docker.io).
+
+
+Versions
+---------------
+Python versions currently supported are:
+* python-3.3
+
+RHEL versions currently supported are:
+* RHEL7
+
+CentOS versions currently supported are:
+* CentOS7
+
+
+Installation
+---------------
+To build Python image, choose between CentOS or RHEL based image:
+*  **RHEL based image**
+
+    To build a rhel-based python-3.3 image, you need to run the build on properly
+    subscribed RHEL machine.
+
+    ```
+    $ git clone https://github.com/openshift/sti-python.git
+    $ cd sti-python
+    $ make build TARGET=rhel7 VERSION=3.3
+    ```
+
+*  **CentOS based image**
+    ```
+    $ git clone https://github.com/openshift/sti-python.git
+    $ cd sti-python
+    $ make build VERSION=3.3
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of Python. Since we are now supporting only version `3.3`,
+you can omit this parameter.**
+
+
+Usage
+---------------------
+To build a simple [python-sample-app](https://github.com/openshift/sti-python/tree/master/3.3/test/setup-test-app) application,
+using standalone [STI](https://github.com/openshift/source-to-image) and then run the
+resulting image with [Docker](http://docker.io) execute:
+
+*  **For RHEL based image**
+    ```
+    $ sti build https://github.com/openshift/sti-python.git --contextDir=3.3/test/setup-test-app/ openshift/python-33-rhel7 python-sample-app
+    $ docker run -p 8080:8080 python-sample-app
+    ```
+
+*  **For CentOS based image**
+    ```
+    $ sti build https://github.com/openshift/sti-python.git --contextDir=3.3/test/setup-test-app/ openshift/python-33-centos7 python-sample-app
+    $ docker run -p 8080:8080 python-sample-app
+    ```
+
+**Accessing the application:**
+```
+$ curl 127.0.0.1:8080
+```
+
+
+Test
+---------------------
+This repository also provides [STI](https://github.com/openshift/source-to-image) test framework,
+which launches tests to check functionality of a simple python application built on top of sti-python image.
+
+User can choose between testing python test application based on RHEL or CentOS image.
+
+*  **RHEL based image**
+
+    This image is not available as automated build in [Docker Index](https://index.docker.io).
```

Comment:
i'm not sure the point in mentioning this since "make test" builds the image anyway.  (I think we have it on other READMEs and we should remove it there too, unless someone can tell me why we have it)